.github/pull_request_template.md

```diff
@@ -36,3 +36,14 @@ Fixes #TODO
 ### Verification
 
 <!-- TODO: Say how you tested your changes. -->
+
+
+### Beyond this PR
+
+<!-- 
+Thank you for submitting this! Have you ever thought of becoming a Reviewer or Approver on the project? 
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are gauging interest in a potential system in which many companies pledge a little bit of time each to help get more people into these roles. See [here](https://github.com/argoproj/argo-workflows/issues/12229) for more information. If you think you or your company might be interested, please add a comment to the issue.
```

Comment:
```suggestion
We are gauging interest in a potential system in which many companies pledge a little bit of time each to help get more people into these roles.
See [#12229](https://github.com/argoproj/argo-workflows/issues/12229) for more information.
If you think you or your company might be interested, please add a comment to the issue.
```

- 1 sentence per line for consistency with the other templates
- also "here" -> `#12229`